### PR TITLE
Implement a mobile version of Pagination

### PR DESCRIPTION
**Pull request by hungdoansy**
# Description
The pagination section in mobile in Campaign is overflowing if the pages are too big. A new version is needed. The new version is shorter in length and has an input to type in which page to go.

### CODE REVIEW

File: src/components/Pagination/PaginationInputOnMobile.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  className?: string
+  page: number
+  lastPage: number
+  setPage: (p: number) => void
+}
+
+const isValidNumber = (s: string): boolean => {
+  const numberRegex = /^\d+$/
+  return !!s.match(numberRegex)
+}
```
Comment (namgold) [inline src/components/Pagination/PaginationInputOnMobile.tsx:17]: why not use `isNaN()`

Comment (hungdoansy) [reply src/components/Pagination/PaginationInputOnMobile.tsx:17]: The string must be a valid integer. isNaN is not enough.
```js
 isNaN('-1.2') == true;
 isNaN('3') == true;
```

Comment (namgold) [reply src/components/Pagination/PaginationInputOnMobile.tsx]: How about `!isNaN(a) && Number.isInteger(Number(a))`

Comment (namgold) [reply src/components/Pagination/PaginationInputOnMobile.tsx]:
```
['-1.2', '-3', '3', '1.2'].map(a => !isNaN(a) && Number.isInteger(Number(a)))
=> [false, true, true, false]
```

Comment (namgold) [reply src/components/Pagination/PaginationInputOnMobile.tsx]: anyway, in this case regex is shorter, I prefer regex more =))

Comment (nguyenhoaidanh) [reply src/components/Pagination/PaginationInputOnMobile.tsx]: I don't think "page" is negative, or float.
Is `Number.isInteger(Number(a))` enought for us ?

Comment (hungdoansy) [reply src/components/Pagination/PaginationInputOnMobile.tsx]: This is to validate user input. We want 1, 2, 3, ... only. In the example above, `3e1` gives `true`.
I think one-line regex is the best so far. 

Btw, `Number.isNaN` should be used instead. Check out https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN, in some first lines.

Comment (namgold) [reply src/components/Pagination/PaginationInputOnMobile.tsx]: `isNaN` and `Number.isNaN` are there for different purposes. Not to the latter gonna replace the first one.

Reference: https://stackoverflow.com/a/33164924/8153505

In this case, `isNaN` match our need over `Number.isNaN`

**Review by namgold — APPROVED** [review]

**Pull request merged**